src/chat/CustomerSupport/Domain.cs

```diff
@@ -1,13 +1,30 @@
 ﻿using System.Text.Json.Serialization;
 using Microsoft.Extensions.AI;
```

Comment:
```suggestion
using System.Text.Json.Serialization;
using Microsoft.Extensions.VectorData;
```
Removing using statements that (tooling says) are unused.

Reply:
Resolved